lit/concepts/internals.lit

```diff
@@ -654,20 +654,40 @@
 
       \italic{Resource Check Containers} are created from the resource type's
       image and are used to check for new versions of a resource.  There will
-      be one per \reference{resource_configs-table}{resource config}
+      be one per \reference{resource_configs-table}{resource config}.
     }
 
     \section{
       \title{Resource Get Container}
 
       \italic{Resource Get Container} are created when a \reference{get-step}
-      is exectuted in a build plan. They are based on the resource type's image
+      is executed in a build plan. They are based on the resource type's image
       and are used to download the bits for a given version of resource.
 
       There will be one per \reference{resource_configs-table}{resource
-      config}
+      config}.
     }
   }
+
+
+  \section{
+    \title{Container Placement Strategies}{container-placement-strategies}
+    As explained above, each step of a build plan is executed inside a container. ATC will distribute containers across the worker cluster depending on the configured strategy. \bold{Default:} \code{volume-locality}
```

Comment:
Recommend making it more terse by starting the sentence with "Each step of a build plan is executed in a container..."

Reply:
I'd also suggest moving the Default: part onto its own line for emphasis, or push that description to the `volume-locality` section itself.

Reply:
done and done.